pkg/component/controller/cplb_unix.go

```diff
@@ -92,6 +93,9 @@ func (k *Keepalived) Start(_ context.Context) error {
 	}
 
 	if len(k.Config.VirtualServers) > 0 {
+		if k.HasEndpointReconciler {
```

Comment:
What about moving that check out the component into the controller command? Otherwise the `HasEndpointReconciler` flag is just some "I will fail to start" flag.